internal/bazel_integration_test/test_runner.js

```diff
@@ -271,6 +271,16 @@ const isWindows = process.platform === 'win32';
 const bazelBinary =
     require.resolve(`${config.bazelBinaryWorkspace}/bazel${isWindows ? '.exe' : ''}`);
 
+if (DEBUG) {
```

Comment:
this should be VERBOSE_LOGS right? DEBUG should affect the build outputs, not affordances we need to see what the build system is doing

Reply:
At one point I had `DEBUG || VERBOSE_LOGS`. In this case `DEBUG` is what bazel_integration_test keys off of to keep the `tmp` folders around instead of having them auto-deleted so it would make sense to print out the `tmp` folder if it is being kept around.

Reply:
hmmm. actually `DEBUG || VERBOSE_LOGS` is more correct as you can CTRL-C the test halfway through with only VERBOSE_LOGS and the tmp folders are not deleted